addons/website/static/src/scss/secondary_variables.scss

```diff
@@ -55,3 +55,4 @@ $o-theme-font: nth($o-theme-fonts, o-website-value('font-number')) !default;
 $o-theme-headings-font: nth($o-theme-fonts, o-website-value('headings-font-number'));
 $o-theme-navbar-font: nth($o-theme-fonts, o-website-value('navbar-font-number'));
 $o-theme-buttons-font: nth($o-theme-fonts, o-website-value('buttons-font-number'));
+$o-theme-number-of-fonts: length($o-theme-fonts);
```

Comment:
We don't need that one, use directly `length($o-theme-fonts)` at the one place it is needed